convene-web/app/views/guides/_identification.md

```diff
@@ -0,0 +1,65 @@
+- [Sign In](#sign-in)
+- [Set Up Your Profile](#set-up-your-profile)
+- [Sign Out](#sign-out)
+
+Everyone in a [Space] is either a [Guest], [Neighbor], [Space Member], or [Space
+Owner]. Until you identify yourself, you are a [Guest]. Some [Spaces] will have
+[Rooms] or [Furniture] that are only for [Neighbors], [Space Members], or [Space
+Owners].
+
+The first time you enter a [Neighborhood], identifying yourself is a two-step
+process: [Sign In](#sign-in) and
+[Set Up Your Profile](#set-up-your-profile).
+
+Once you've set up your Profile for a [Neighborhood], it will be used in
+all of that [Neighborhood's] [Spaces].
+
+### Sign In
+
+To switch from being a [Guest] to being a [Neighbor], [Space Member],
+or [Space Owner]; sign in using the [Personal Navigation Menu].
+
+You'll be asked to provide your email address, and follow a link to confirm you
+are who you say you are.
+
+### Set Up Your Profile
+
+Your _Profile_ is what other people know about you. This includes:
+
+1. Your _Name_, so folks know what to call ya. This may be a pseudonym,
+   nick-name, or even your full title; like "Dr. Cornelius Reginald The Third"
+   or "Doc" or "Corny Reg" or... You get the picture.
+2. Your _Pronouns_, (optional) which is how people refer to you when you're not
+   around. Defaults to "they"
+3. Your _Avatar_, (optional) so people can recognize you at a glance. This could
+   be a real photo, a character, some art. The important part is picking one
+   that is meaningful _to you_. Defaults to a randomly generated unique pattern.
+
+Profiles _must not_ include slurs or other dehumanizing expressions. Each
+[Neighborhood] decides what other kinds of expression is and is not acceptable
+for them.
+
+### Sign Out
+
+When you are done in a [Space] you can stay signed in, or you can sign out so
+that you start future visits to the space are as a [Guest].
+
+To sign out, use the [Personal Navigation Menu].
+
+[internal rooms]: ./rooms#internal-rooms
+[neighborhood]: ./neighborhoods
+[spaces]: ./spaces
+[space]: ./spaces
+[neighbors]: ./people#neighbors
+[neighbor]: ./people#neighbors
+[space members]: ./people#space-members
+[space member]: ./people#space-members
+[space owners]: ./people#space-owners
+[space owner]: ./people#space-owners
+[guests]: ./people#guests
+[guest]: ./people#guests
+[sign in]: #sign-in
+[sign out]: #sign-out
+[personal navigation menu]: ./getting_around#personal-navigation-menu
+[rooms]: ./rooms
+[furniture]: ./furniture
```

Comment:
As the guides grow, it would be neat if this list of link definitions could somehow be shared across pages (like a partial, but in markdown).